doc/tutorials/enable_s5.rst

```diff
@@ -216,12 +217,12 @@ How to Test
            Active: active (running) since Tue 2019-09-10 07:15:06 UTC; 1min 11s ago
            Main PID: 840 (life_mngr)
 
-   .. note:: For WaaG, we need to close ``windbg`` by using the ``bcdedit /set debug off`` command
+   .. note:: For WaaG, close ``windbg`` by using the ``bcdedit /set debug off`` command
       IF you executed the ``bcdedit /set debug on`` when you set up the WaaG, because it occupies the ``COM2``.
 
 #. Use the ``acrnctl stop`` command on the Service VM to trigger S5 to the User VMs:
 
-   .. code-block:: console
+   .. code-block:: none
```

Comment:
As @gvancuts would ask, let's get rid of these command prompts.  (Command lines with a ``#`` should use ``sudo``)